InteractiveHtmlBom/web/pep.js

```diff
@@ -0,0 +1,1473 @@
+/*!
```

Comment:
Please use minified version
https://code.jquery.com/pep/0.4.3/pep.min.js
With a newline at the end of file

Preferably remove comments from it too, I have no idea why they kept them in the minified version.

Reply:
I should have mentioned: don't remove the top comment with copyright/license and link to original source.

Reply:
Resolved in latest commit